### DISCUSSION TITLE
How to initialize Structural case with thermal expension

### DISCUSSION THREAD

**Discussion by mnabideltares** [KratosMultiphysics/Kratos#11190] [General]
Hi,

We would like to solve structural element. However, we also want to include initial temperature, and hence, there is initial thermal expension eps_0.
As far as I know, Strurtural application is able to do it but activating it in the json input file. However, we are not able to find the proper keyword(s) related to this.

We will be very thankful if somebody give us a hint.

Regards

**Comment by AlejandroCornejo** [comment]
hi! 
if you know the values of these initial strains you can directly use the `initial_state` capability. Basically you need to use the `set_initial_state_process` which will fill the initial strain value and, if the constitutive law has implemented the initial_state addition (the elastic CL in the CLApp do it) you'll be accounting for that.

**Reply by AlejandroCornejo** [reply]
look what I did back then: https://github.com/KratosMultiphysics/Kratos/pull/7663

**Reply by mnabideltares** [reply]
@AlejandroCornejo , Thank you very much. It is more clear to me now.

**Comment by AlejandroCornejo** [comment]
Closing since seems to be solved. 

@mnabideltares do not hesitate to contact me directly or reopen this issue again if you need any help/guidance
